src/ui/components/ShowMoreCard/index.js

```diff
@@ -11,8 +11,9 @@ import Card from 'ui/components/Card';
 
 import './ShowMoreCard.scss';
 
-
-const MAX_HEIGHT = 100;
+// This refers to height of `Card-contents` CSS class,
+// beyond which it will add read more link.
+export const MAX_HEIGHT = 145;
```

Comment:
Why not having `150` actually? I cannot get the issue mentioned in #3148 fixed with this value.